streams/src/main/java/org/apache/kafka/streams/kstream/internals/graph/KTableKTableJoinNode.java

```diff
@@ -33,20 +34,27 @@
  */
 public class KTableKTableJoinNode<K, V1, V2, VR> extends BaseJoinProcessorNode<K, Change<V1>, Change<V2>, Change<VR>> {
 
+    private final Serde<K> keySerde;
+    private final Serde<VR> valueSerde;
     private final String[] joinThisStoreNames;
     private final String[] joinOtherStoreNames;
-    private final MaterializedInternal<K, VR, KeyValueStore<Bytes, byte[]>> materializedInternal;
+    // The name of queryableStore, which stores the join results.
+    private final String queryableStoreName;
+    private final StoreBuilder<KeyValueStore<K, VR>> storeBuilder;
```

Comment:
Should this get inlined with the types to `StoreBuilder<KeyValueStore<Bytes, byte[]>>`?
\cc @guozhangwang

Reply:
Hmm why can we do that? The storeBuilder returned from the `KeyValueStoreMaterializer` is a fully-wrapped one and not an inner bytes-store, right?

Reply:
Ah, yes that's correct, @dongjinleekr ignore my comment here.